service/static/.land-registry-elements/assets/javascripts/govuk-template.js

```diff
@@ -77,20 +77,6 @@
 (function() {
   "use strict"
 
-  // fix for printing bug in Windows Safari
```

Comment:
@russwillis This has come from upstream in the gov uk template. See this commit: https://github.com/alphagov/govuk_template/commit/4348f4319396369b65015544c5d41bf3ca704067